src/jio-footer.ts

```diff
@@ -69,16 +69,13 @@ export class Footer extends LitElement {
    reportAProblemTemplate = "";
 
    override render() {
-      // Check if the current page is /download or /download/mirrors
-      const isADownloadsPage = this.sourcePath.includes('/download/') || this.sourcePath.includes('/download/mirrors/');
```

Comment:
this feels like a very implementation hacked into something global
why not just add a property that can be overridden

```
@property({ type: Boolean })
skipReportIssue = false;
```

I'll be honest, i don't know why a function inside of html would do anything different than before. If its a weird behavior that html will re-render more often than render(), then you should specifically add a test to make sure a future upgrade doesn't break this niche behavior.

Reply:
The fact is that the function was not re-rendered after the first time, so we need to bring it outside to make sure it re-renders afterwards as well.

Reply:
I did check the documentation and my implementation is the one they suggested would work like a React hook given the context

Reply:
Thanks @halkeye! Let me see if I can find someone else to review this PR

Reply:
For a reference: https://codelabs.developers.google.com/codelabs/lit-2-for-react-devs#6

Reply:
I did apply the suggested `@property` change